bin/zsnap

```diff
@@ -39,15 +40,27 @@ else:
 set_verbose(opts.verbose)
 
 snapshot_prefix = opts.prefix
-snapshot_postfix = lambda: time.strftime(opts.timeformat)
+snapshot_postfix = lambda: time.strftime(opts.timeformat,time.gmtime())
```

Comment:
There is an extraneous time.gmtime() call here -- this should probably only be used if a command line option (that defaults to false) to generate timestamps in UTC is presented by the user.  We can't change the default because it would break several deployments.